index.html

```diff
@@ -123,17 +143,79 @@
     // SCALES
 
     x = d3.scale.linear()
-      .domain([0, d3.max(data, function (d) { return d.unemployment; })])
+      .domain(d3.extent(data, function (a) {return a.costs; }))
       .range([0, width])
       .nice();
 
     y = d3.scale.linear()
-      .domain(d3.extent(data, function (d) { return d.inflation; }))
+      .domain(d3.extent(data, function (a) {return a.visits; }))
       .range([height, 0])
       .nice();
 
     // AXES
 
+    var formatNumber = d3.format(".0f");
+
+    var xAxis = d3.svg.axis()
+                  .scale(x)
+                  .tickFormat(formatCurrency)
+                  .tickSize(height)
+                  .orient("bottom"); 
+
+    svg.append("g")
+       .attr("class", "x axis")
+       //.attr("transform", "translate(0," + (height) + ")")
+       .call(xAxis)
+      
+       .append("text")
+       .attr("x", 250)
+       .attr("y", 580)
+       .style("text-anchor", "start")
+       .text("Standardized, Risk-Adjusted Per Capita Medicare Spending");
+
+    var gx = svg.append("g")
+                .attr("class", "x axis")
+                .call(xAxis);
+
+    gx.selectAll("g").filter(function(a) {return a;})
+      .classed("minor", true);
+
+    var yAxis = d3.svg.axis()
+                  .scale(y)
+                  .tickSize(width)
+                  .orient("right");
+
+    svg.append("g")
+       .attr("class", "y axis")
+       .attr("transform","translate(0,0)")
+       .call(yAxis)
+
+       .append("text")
+       .attr("transform", "rotate(-90)")
+       .attr("x", -300)
+       .attr("y", -60)
+       .attr("dy", ".9em")
+       .style("text-anchor", "start")
+       .text("ER Visits per 100K");
+
+    var gy = svg.append("g")
+                .attr("class", "y axis")
+                .call(yAxis);
+
+      gy.selectAll("g").filter(function(a) {return a;})
+        .classed("minor", true);
+
+      gy.selectAll("text")
+        .attr("x", -30)
+        .attr("dy", -4);
```

Comment:
Looks like you've figured out how to move the points over. But if they appear to the left of the axis, we'd expect these numbers to be right-aligned. This would indicate that maybe we should instead change the `axis.orient()` on line 186.